HW3/Menues/Database/Books.cs

```diff
@@ -50,7 +50,7 @@ public void Delete(SqlConnection connection)
             string nameForDelete = Console.ReadLine();
 
             string query;
-            if (string.IsNullOrEmpty(nameForDelete))
+            if (string.IsNullOrEmpty(nameForDelete)) // can be shorter, using ?: ternary operator
             {
                 query = $"delete from Books";
```

Comment:
Better write commands in upper-case (DELETE, INSERT etc)

Reply:
Done